test/Microsoft.IdentityModel.TestUtils/CertificateHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.IdentityModel.TestUtils
+{
+    /// <summary>
+    /// Helper class to load X509Certificate2 from byte array.
+    /// </summary>
+    public class CertificateHelper
```

Comment:
I think it's okay to share internals with our test libraries, since tests reference the projects as source, so this class may not be needed.

Reply:
I'm going to keep the test public helper class as there were issues with sharing internals with one of the test projects.